igdiscover/igdiscover.yaml

```diff
@@ -204,7 +204,7 @@ germline_filter:
   cdr3_shared_ratio: 0.8     # Maximum allowed CDR3_shared_ratio
   unique_d_ratio: 0.3        # Minimum Ds_exact ratio between alleles
   unique_d_threshold: 10     # Check Ds_exact ratio only if highest-expressed allele has at least this Ds_exact count
-
+  max_db_diff: -1            # Maximum allowed database differences (negative is disabled)
```

Comment:
This should probably be `maximum_database_difference`. It’s a bit long, but everything else is spelled out as well, and users don’t need to type it themselves when they edit the configuration file.

The setting should also be added to the `pre_germline_filter` section.

Reply:
Now present in both pregermline and germline filter.